src/command.coffee

```diff
@@ -444,6 +444,9 @@ forkNode = ->
     cwd:        process.cwd()
     env:        process.env
     stdio:      [0, 1, 2]
+  ['SIGINT', 'SIGTERM'].forEach (signal) ->
```

Comment:
Our prefer style is to use `for…in`:

```coffee
for signal in ['SIGINT', 'SIGTERM']
```

This compiles into a traditional `for` loop, without a new function or scope.